pkg/backend/filestate/store.go

```diff
@@ -19,6 +19,7 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	pkgPath "path"
```

Comment:
@abhinav I prefer to have the imported path package renamed to `pkgPath` instead of renaming all `path` variable in this file (to something like `p` or `pt`).

Simply, due to lesser code changes.

LMK if you have concerns related this.

Reply:
Thanks a lot for the updates, @neverbeenthisweeb.
If you wouldn't mind, could you please use just "path" for it without the qualified import? 
We usually avoid named imports unless there's a conflict with another import.

```suggestion
	"path"
```

I agree that the name `path` is a bit annoying and likely to conflict with a variable, but it should be fine in this case.
We have only one instance:

```
path := path.Join(...)
```

This will compile just fine. Only on the following line will it make the `path` package inaccessible:

```go
path := path.Join(...)
x = path.Join(...) // error: path is a string
```

For the rare case where that's a problem, I'm okay with us having to resolve the conflict by naming the variable differently.

The change looks good otherwise!

Reply:
I see. OK, I will do the update.

Reply:
Updated https://github.com/pulumi/pulumi/pull/12798/commits/b1672afee654b5200c6fdd3cee28bcaa40acf9e5